explorer/client/src/components/search/SearchResults.tsx

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { type ResultType } from './SearchResultType';
+
+import styles from './SearchResults.module.css';
+
+function SearchResults({
+    result,
+    resultIndex,
+    setResultIndex,
+    optionClick,
+}: {
+    result: ResultType[] | null;
+    resultIndex: number;
+    setResultIndex: (index: number) => void;
+    optionClick: (el: ResultType) => () => void;
+}) {
+    const convertLabel = (category: string) => {
```

Comment:
An easier way to implement this would probably just be an object:
```ts
const categoryLabels = {
  objects: 'Object',
  transactions: 'transaction',
  addresses: 'address',
};
```

Reply:
Done.